src/scripts/main.js

```diff
@@ -1 +1,14 @@
 'use strict';
+
+const galleryList = document.querySelector('.gallery__list');
+const mainImg = document.querySelector('#largeImg');
+
+galleryList.addEventListener('click', (event) => {
```

Comment:
Try to fix this error
![image](https://user-images.githubusercontent.com/56752562/228471683-2d16e539-ed24-4a26-b696-7de68fa466fc.png)

Reply:
I can rename "event", but i dont get why is linter yelling at me.